async function checkUser() {
    try {
        const user = await Auth.currentAuthenticatedUser();
        console.log("User: ", user);
    } catch (error) {
        console.log("User is not sign in");
    }
}
